### PR TITLE
Add content + tools template - ecosystem vulnerability tools 

### PR DESCRIPTION
**Pull request by nickumia-reisys**
Related to
- https://github.com/GSA/data.gov/issues/3891

Replaces
- https://github.com/GSA/datagov-website/blob/main/www.data.gov/tools.md

New Additions:
- Revive modal feature by including bootstrap
- Abstract modal feature into 11ty template
- Convert `ecosystem-vulnerability-tools` to use the new template

References:
- https://getbootstrap.com/docs/4.0/components/modal/#scrolling-long-content
- https://getbootstrap.com/docs/4.6/getting-started/download/#jsdelivr
- https://www.11ty.dev/docs/pages-from-data/

### CODE REVIEW

File: posts/2014-11-14-ecosystem-vulnerability-tools.md [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+**Name:** Integrated Taxonomic Informatin System (IT IS)
+**URL:** <http://www.itis.gov/tools.html>
+**Summary:**
+The ITIS database currently provides tools to compare species names with their hierarchical classifications. Agencies that typically cannot afford to maintain taxonomic data will have access to high quality taxonomic information through ITIS.
+**How it works:**
+ITIS currently includes two tools. The first is a prototype to compare taxonomy and nomenclature. The Taxonomic Workbench is a Windows-based software tool used for editing and manipulating taxonomic data for submission into the ITIS online system. This software is used for entering new data into the ITIS database, as well as updating records already in the online system.
+**Application:**
+Federal agencies have the ability to use the ITIS tool in order to promote improved coordination of efforts among federal agencies, thereby increasing productivity and saving resources.
+**Preview:**
+[![](https://s3-us-gov-west-1.amazonaws.com/cg-0817d6e3-93c4-4de8-8b32-da6919464e61/IntegratedTaxonomicInformatinSystem.png)](http://www.itis.gov/tools.html)
```
Comment (robert-bryson) [inline posts/2014-11-14-ecosystem-vulnerability-tools.md:71]: I think this is failing cause it's missing an [alt text](https://www.markdownguide.org/basic-syntax/#images-1) in the `[]` here.

File: posts/2014-11-14-ecosystem-vulnerability-tools.md [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+**URL:** <http://gapanalysis.usgs.gov/viewers/>
+**Summary:**
+The Gap Analysis Program has created interactive maps for viewing its datasets. The Land Cover Viewer displays data on vegetation and land use patterns. The PAD-US Viewer displays federal and state conservation lands. The Species Viewer displays species range.
+**How it works:**
+Using the website, users can download data or view data for vegetation and land use patterns, federal and state conservation lands and species ranges. To conduct a gap analysis, the three data layers are overlaid.
+**Application:**
+Users can map three data layers that are provided – A Gap Analysis consists of mapping three data layers — land cover, predicted distributions of vertebrate species, and a stewardship layer. This data is then assessed to determine how much of a target species’ (plant or animal) habitat is in conserved areas. From this assessment, planning decisions can be made about whether further protection is merited.
+**Preview:**
+[![](https://s3.amazonaws.com/bsp-ocsit-prod-east-appdata/datagov/wordpress/2014/11/NationalGapAnalysisProgram.png)](http://gapanalysis.usgs.gov/viewers/)
```
Comment (robert-bryson) [inline posts/2014-11-14-ecosystem-vulnerability-tools.md:85]: And here

File: posts/2014-11-14-ecosystem-vulnerability-tools.md [hunk 3]
```diff
@@ -0,0 +1,333 @@
...
+**Name:** USFS Climate Change Atlas
+**URL:** <http://www.fs.fed.us/nrs/atlas/>
+**Summary:**
+The Climate Change Atlas documents the current and possible future distribution of 134 tree species and 147 bird species in the Eastern United States and gives detailed information on environmental characteristics defining these distributions.
+**How it works:**
+Users can see which factors (e.g. temperature, elevation, soil type) help to drive species distributions, offering some guidance on species sensitivity to large-scale climate differences. The Atlases supply maps and summary numerical data that show how the suitable habitat of each species is projected to change under three different climate models, for both high and low emissions scenarios. Maps can be viewed via the Atlas online interface, or with the program, Google Earth.
+**Application:**
+The Atlas can be used in the development of large scale projections of species responses to climate change, which can inform long-term planning.
+**Preview:**
+[![](https://s3.amazonaws.com/bsp-ocsit-prod-east-appdata/datagov/wordpress/2014/11/USFSClimateChangeAtlas.png)](http://www.fs.fed.us/nrs/atlas/)
```
Comment (robert-bryson) [inline posts/2014-11-14-ecosystem-vulnerability-tools.md:100]: And here

File: posts/2014-11-14-ecosystem-vulnerability-tools.md [hunk 4]
```diff
@@ -0,0 +1,333 @@
...
+**Name:** COMET-Farm
+**URL:** <http://cometfarm.nrel.colostate.edu/>
+**Summary:**
+COMET-FARM estimates the ‘carbon footprint’ for all or part of your farm/ranch operation and allows you to evaluate different options, which you select, for reducing GHG emissions and sequestering more carbon.
+**How it works:**
+The tool uses detailed spatially-explicit data on climate and soil conditions for your location and allows you to enter detailed information for your field and livestock operations. Because of this, it is able to produce an accurate estimate tailored to your specific situation.
+**Application:**
+Farm/ranch operation managers can use this tool to estimate their carbon footprints and explore options for reducing GHG emissions and sequestering greater levels of carbon.
+**Preview:**
+[![](https://s3.amazonaws.com/bsp-ocsit-prod-east-appdata/datagov/wordpress/2014/11/COMETFarm.png)](http://cometfarm.nrel.colostate.edu/)
```
Comment (robert-bryson) [inline posts/2014-11-14-ecosystem-vulnerability-tools.md:117]: here

File: posts/2014-11-14-ecosystem-vulnerability-tools.md [hunk 5]
```diff
@@ -0,0 +1,333 @@
...
+**Name:** ROE Carbon Storage in Forests
+**URL:** <http://cfpub.epa.gov/roe/indicator.cfm?i=86#1>
+**Summary:**
+The Report on the Environment (ROE) presents the best available indicators of national trends in the environment and human health. One of these indicators is carbon storage in forests, and the ROE Carbon Storage in Forests tool can be used to explore the trends related to this indicator.
+**How it works:**
+The tool is divided into the following thee “exhibits”: a graph of total forest biomass in the U.S., by forest component, 1990-2013; a map of forest biomass in U.S. counties per square mile of land, 2013; and a map of percent change in carbon stored in U.S. forests, by county, 1990-2013
+**Application:**
+This tool can be used by a wide variety of people, including decision makers, program planners, scientists, researchers, students, and the general public.
+**Preview:**
+[![](https://s3.amazonaws.com/bsp-ocsit-prod-east-appdata/datagov/wordpress/2014/11/ROECarbonStorageinForests.png)](http://cfpub.epa.gov/roe/indicator.cfm?i=86#1)
```
Comment (robert-bryson) [inline posts/2014-11-14-ecosystem-vulnerability-tools.md:133]: here

File: posts/2014-11-14-ecosystem-vulnerability-tools.md [hunk 6]
```diff
@@ -0,0 +1,333 @@
...
+2. Uses remote sensing input data, existing resource and soil inventories, climate histories, and measurements made by a national network of streamgages
+3. Applies a set of future land- and climate-change scenarios to the assessment to estimate a range of carbon stocks and sequestration rates in ecosystems
+**How it works:**
+The LandCarbon datasets provide information on the changes in carbon stocks and fluxes in all the major ecosystems, and on the major natural and anthropogenic processes that control carbon cycling (such as climate change, land use, and wildland fires). Information is provided as a series of thematic maps chronicling carbon stocks and their controlling processes.
+**Application:**
+The LandCarbon project maps how much carbon is sequestered in ecosystems using stream gauge, soil, and natural-resource inventory data, remote sensing techniques, and computer models. A number of data products and visualization tools can be used to further explore the information.
+**Preview:**
+[![](https://s3.amazonaws.com/bsp-ocsit-prod-east-appdata/datagov/wordpress/2014/11/LandCarbon.png)](http://landcarbon.org/tools/)
```
Comment (robert-bryson) [inline posts/2014-11-14-ecosystem-vulnerability-tools.md:155]: here

File: posts/2014-11-14-ecosystem-vulnerability-tools.md [hunk 7]
```diff
@@ -0,0 +1,333 @@
...
+**Name:** FHTET Pest Portal
+**URL:** <http://foresthealth.fs.usda.gov/portal>
+**Summary:**
+The portal offers a number of datasets and near real time information on forest insect and disease conditions.
+**How it works:**
+Data from the portal are collected from the air via standard aerial survey, from ground surveys or from remote sensing analysis. Data is “rounded up” from all Forest Service regions in the fall of each year.
+**Application:**
+Forest managers can use these tools to understand oncoming threats to forests from disease and invasive species.
+**Preview:**
+[![](https://s3.amazonaws.com/bsp-ocsit-prod-east-appdata/datagov/wordpress/2014/11/FHTETPestPortal.png)](http://foresthealth.fs.usda.gov/portal)
```
Comment (robert-bryson) [inline posts/2014-11-14-ecosystem-vulnerability-tools.md:171]: here

File: posts/2014-11-14-ecosystem-vulnerability-tools.md [hunk 8]
```diff
@@ -0,0 +1,333 @@
...
+**How it works:**
+EnviroAtlas is a collaborative project and incorporates data from federal, state, community, and non-governmental organizations. The data helps users to understand the services that ecosystems provide.
+**Application:**
+Using EnviroAtlas, many types of users can access, view, and analyze diverse information to better understand how various decisions can affect an array of ecological and human health outcomes. EnviroAtlas has already been used by a variety of decision makers, including urban and community planners.
+**Preview:**
+[![](https://s3.amazonaws.com/bsp-ocsit-prod-east-appdata/datagov/wordpress/2014/11/EnviroAtlasDownloadableGISTools.png)](http://enviroatlas.epa.gov/enviroatlas/Tools/index.html)
```
Comment (robert-bryson) [inline posts/2014-11-14-ecosystem-vulnerability-tools.md:187]: ^^

File: posts/2014-11-14-ecosystem-vulnerability-tools.md [hunk 9]
```diff
@@ -0,0 +1,333 @@
...
+**How it works:**
+The National Land Cover Database Evaluation Visualization and Analysis (NLCD EVA) tool provides user-friendly access to national and state land cover and land cover change information developed through the Multi-Resolution Land Characteristics (MRLC) Consortium. NLCD EVA eliminates the need for desktop geographic information system software, or advanced technical expertise, by processing NLCD data for the user and providing easy access to that distilled information.
+**Application:**
+U.S. and international producers, researchers, and the public can access geospatial information on the location and extent of land cover/use and its changes
+**Preview:**
+[![](https://s3.amazonaws.com/bsp-ocsit-prod-east-appdata/datagov/wordpress/2014/11/NationalLandCoverDatabaseEvaluationVisualizationandAnalysisTool.png)](http://www.mrlc.gov/eva/)
```
Comment (robert-bryson) [inline posts/2014-11-14-ecosystem-vulnerability-tools.md:202]: ^^

File: posts/2014-11-14-ecosystem-vulnerability-tools.md [hunk 10]
```diff
@@ -0,0 +1,333 @@
...
+**Name:** Coastal Change Analysis Project (C-CAP)
+**URL:** <https://coast.noaa.gov/digitalcoast/tools/lca>
+**Summary:**
+The Coastal Change Analysis Program (C-CAP) produces a nationally standardized database of land cover information (developed using remotely sensed imagery) for the coastal regions of the U.S. C-CAP products provide inventories of coastal intertidal areas, wetlands, and adjacent uplands. Datasets can be filtered by the user using various categories.
+**How it works:**
+C-CAP utilizes various sources of data, including aerial and satellite imagery. It is produced using standard data sources and procedures that include extensive field sampling, validation, and standard quality control review procedures. The data registry can be filtered by state, theme, offshore region or service type.
+**Application:**
+The tool can be used by community and regional planners to inform coastal development.
+**Preview:**
+[![](https://s3.amazonaws.com/bsp-ocsit-prod-east-appdata/datagov/wordpress/2014/11/CoastalChangeAnalysisProject.png)](https://coast.noaa.gov/digitalcoast/tools/lca)
```
Comment (robert-bryson) [inline posts/2014-11-14-ecosystem-vulnerability-tools.md:218]: ^^

File: posts/2014-11-14-ecosystem-vulnerability-tools.md [hunk 11]
```diff
@@ -0,0 +1,333 @@
...
+**Summary:**
+Environmental Sensitivity Index (ESI) maps provide a concise summary of coastal resources that are at risk if an oil spill occurs nearby. Examples of at-risk resources include biological resources (such as birds and shellfish beds), sensitive shorelines (such as marshes and tidal flats), and human-use resources (such as public beaches and parks).
+**How it works:**
+Uses GIS techniques to integrate maps of a region with geographically-referenced biological resources,human-use resources, and ESI-classified shorelines that are ranked based on their sensitivity to oiling.
+**Application:**
+Oil spill responders can use this tool to reduce environmental consequences of oil spills and clean up efforts. Coastal zone planners can also use the tool – before a spill happens—to identify vulnerable locations, establish protection priorities, and identify cleanup strategies.
+**Preview:**
+[![](https://s3.amazonaws.com/bsp-ocsit-prod-east-appdata/datagov/wordpress/2014/11/EnvironmentalSensitivityIndex.png)](http://response.restoration.noaa.gov/maps-and-spatial-data/environmental-sensitivity-index-esi-maps.html)
```
Comment (robert-bryson) [inline posts/2014-11-14-ecosystem-vulnerability-tools.md:234]: ^^

File: posts/2014-11-14-ecosystem-vulnerability-tools.md [hunk 12]
```diff
@@ -0,0 +1,333 @@
...
+**Name:** NOAA’s Coral Reef Information System (CoRIS)
+**URL:** <http://www.coris.noaa.gov/>
+**Summary:**
+CoRIS is a portal that allows users to search for coral reef data and information.
+**How it works:**
+Users can search for a variety of data using the online portal. Data is available in a variety of formats including .GEORSS, .HTML, .CSV, .ATOM, etc.
+**Application:**
+CoRIS can be used by a variety of groups. Scientists and researchers can use it to obtain data for further study.
+**Preview:**
+[![](https://s3.amazonaws.com/bsp-ocsit-prod-east-appdata/datagov/wordpress/2014/11/NOAAsCoralReefInformationSystem.png)](http://www.coris.noaa.gov/)
```
Comment (robert-bryson) [inline posts/2014-11-14-ecosystem-vulnerability-tools.md:250]: ^^

File: posts/2014-11-14-ecosystem-vulnerability-tools.md [hunk 13]
```diff
@@ -0,0 +1,333 @@
...
+**Name:** Ocean Biogeographic Information System (OBIS)
+**URL:** <http://www.usgs.gov/obis-usa/>
+**Summary:**
+OBIS-USA is part of an international repository (OBIS) that is coordinated by UNESCO. It contains various data related to marine species occurrence.
+**How it works:**
+The data contained in OBIS-USA may be used in two different ways. First, using the data dashboard, users can visually examine the full collection of data down to the record level. Second, the Biogeographic Analysis Tool provides an analysis or summary of the biodiversity for defined geographic regions.
+**Application:**
+The OBIS portal has a large spectrum of users: researchers, fishery scientists and managers, policy maker, educators, amateur naturalists, environmental NGO, consultants, nature conservation organizations, and students.
+**Preview:**
+[![](https://s3.amazonaws.com/bsp-ocsit-prod-east-appdata/datagov/wordpress/2014/11/OceanBiogeographicInformationSystem.png)](http://www.usgs.gov/obis-usa/)
```
Comment (robert-bryson) [inline posts/2014-11-14-ecosystem-vulnerability-tools.md:266]: ^^

File: posts/2014-11-14-ecosystem-vulnerability-tools.md [hunk 14]
```diff
@@ -0,0 +1,333 @@
...
+**Name:** National Integrated Drought Information System (NIDIS) Drought Portal
+**URL:** <http://www.drought.gov/drought/>
+**Summary:**
+NIDIS is an interagency effort that provides up-to-date information on drought across the nation. The Drought Portal provides various information and tools to help decision makers monitor conditions and prepare for onsets of drought.
+**How it works:**
+Users can search for a variety of up-to-date information on drought across the nation, displayed in the form of briefings, charts and maps. Maps include the U.S. Drought Monitor, an outlook of seasonal drought, reports of drought impacts, and wildfire risks. Users can also find information on various drought indices. The Drought Portal also links to state and regional information.
+**Application:**
+Water resource managers and community and urban planners can use the information to prepare for onsets of drought and to create long-term drought plans.
+**Preview:**
+[![](https://s3.amazonaws.com/bsp-ocsit-prod-east-appdata/datagov/wordpress/2014/11/NationalIntegratedDroughtInformationSystemDroughtPortal.png)](http://www.drought.gov/drought/)
```
Comment (robert-bryson) [inline posts/2014-11-14-ecosystem-vulnerability-tools.md:282]: ^^

File: posts/2014-11-14-ecosystem-vulnerability-tools.md [hunk 15]
```diff
@@ -0,0 +1,333 @@
...
+**URL:** [**https://www.ncdc.noaa.gov/paleo-search/reports/all?dataTypeId=59&search=true**](https://www.ncdc.noaa.gov/paleo-search/reports/all?dataTypeId=59&search=true)
+**Summary:**
+A number of paleoclimatology datasets are provided through NOAA, which are derived from tree rings, ice cores, corals, and ocean and lake sediments. These datasets help to extend our weather and climate data record back hundreds to millions of years. The data helps us to understand natural climate variability and future climate change. NOAA provides a number of software packages to help analyze climate anomalies using this data.
+**How it works:**
+Software packages are contributed by the paleoclimatological community. The packages are provided on an “as-is” basis, and additional support cannot be provided by NOAA.
+**Application:**
+The software packages are primarily aimed at researchers and scientists who can use the software to further analyze the vast area of paleoclimatological data.
+**Preview:**
+[![](https://s3.amazonaws.com/bsp-ocsit-prod-east-appdata/datagov/wordpress/2017/05/paleo_climate.png)](https://www.ncdc.noaa.gov/paleo-search/reports/all?dataTypeId=59&search=true)
```
Comment (robert-bryson) [inline posts/2014-11-14-ecosystem-vulnerability-tools.md:297]: ^^

File: posts/2014-11-14-ecosystem-vulnerability-tools.md [hunk 16]
```diff
@@ -0,0 +1,333 @@
...
+**URL:**
+<http://www.wcc.nrcs.usda.gov/scan/>
+**Summary:**
+The SCAN tools are a collection of soil-climate data from stations around the United States which can be viewed interactively by users.
+**How it works:**
+SCAN uses a variety of methods to transmit remote station data. Users can then view data for each station using an interactive map tool.
+**Application:**
+SCAN can be used by planners to monitor the onset of drought, watershed health, and wetlands.
+**Preview:**
+[![](https://s3.amazonaws.com/bsp-ocsit-prod-east-appdata/datagov/wordpress/2014/11/SoilClimateAnalysisNetwork.png)](http://www.wcc.nrcs.usda.gov/scan/)
```
Comment (robert-bryson) [inline posts/2014-11-14-ecosystem-vulnerability-tools.md:316]: ^^

File: posts/2014-11-14-ecosystem-vulnerability-tools.md [hunk 17]
```diff
@@ -0,0 +1,333 @@
...
+**Summary:**
+This website provides data and links to LANDFIRE data, articles/reports, data descriptions, databases, LANDFIRE videos and tutorials, images/maps, websites, and decision making/informational tools. These resources are useful for practitioners in the fields of fire and natural resource management, vegetation, habitat, carbon/climate, research and science, as well as industry/private sector users and teachers.
+**How it works:**
+LANDFIRE products are designed to be used at a landscape-scales. LANDFIRE provides over 20 national geo-spatial layers (e.g. vegetation, fuel, disturbance, etc.), databases and ecological models.
+**Application:**
+LANDFIRE data products provide data for landscape assessment, analysis, and management. Data and information serve as important data sets in decision support with efforts such as identification of areas with similar characteristics, prioritization exercises, modeling capacity and potential, and improving collaboration between landowners with common data sets and analytics.
+**Preview:**
+[![](https://s3.amazonaws.com/bsp-ocsit-prod-east-appdata/datagov/wordpress/2014/11/Landfire.png)](http://www.landfire.gov)
```
Comment (robert-bryson) [inline posts/2014-11-14-ecosystem-vulnerability-tools.md:332]: ^^

**Review by robert-bryson — CHANGES_REQUESTED** [review]

File: posts/2014-11-14-ecosystem-vulnerability-tools.md (hunk 1 above)
Comment (Jin-Sun-tts) [reply posts/2014-11-14-ecosystem-vulnerability-tools.md]: fixed

File: posts/2014-11-14-ecosystem-vulnerability-tools.md (hunk 3 above)
Comment (Jin-Sun-tts) [reply posts/2014-11-14-ecosystem-vulnerability-tools.md]: fixed

File: posts/2014-11-14-ecosystem-vulnerability-tools.md (hunk 5 above)
Comment (Jin-Sun-tts) [reply posts/2014-11-14-ecosystem-vulnerability-tools.md]: fixed

File: posts/2014-11-14-ecosystem-vulnerability-tools.md (hunk 14 above)
Comment (Jin-Sun-tts) [reply posts/2014-11-14-ecosystem-vulnerability-tools.md]: fixed

File: posts/2014-11-14-ecosystem-vulnerability-tools.md (hunk 16 above)
Comment (Jin-Sun-tts) [reply posts/2014-11-14-ecosystem-vulnerability-tools.md]: fixed

File: posts/2014-11-14-ecosystem-vulnerability-tools.md (hunk 17 above)
Comment (Jin-Sun-tts) [reply posts/2014-11-14-ecosystem-vulnerability-tools.md]: fixed

File: posts/2014-11-14-ecosystem-vulnerability-tools.md (hunk 2 above)
Comment (Jin-Sun-tts) [reply posts/2014-11-14-ecosystem-vulnerability-tools.md]: fixed

File: posts/2014-11-14-ecosystem-vulnerability-tools.md (hunk 4 above)
Comment (Jin-Sun-tts) [reply posts/2014-11-14-ecosystem-vulnerability-tools.md]: fixed

File: posts/2014-11-14-ecosystem-vulnerability-tools.md (hunk 6 above)
Comment (Jin-Sun-tts) [reply posts/2014-11-14-ecosystem-vulnerability-tools.md]: fixed

File: posts/2014-11-14-ecosystem-vulnerability-tools.md (hunk 7 above)
Comment (Jin-Sun-tts) [reply posts/2014-11-14-ecosystem-vulnerability-tools.md]: fixed

File: posts/2014-11-14-ecosystem-vulnerability-tools.md (hunk 8 above)
Comment (Jin-Sun-tts) [reply posts/2014-11-14-ecosystem-vulnerability-tools.md]: fixed

File: posts/2014-11-14-ecosystem-vulnerability-tools.md (hunk 9 above)
Comment (Jin-Sun-tts) [reply posts/2014-11-14-ecosystem-vulnerability-tools.md]: fixed

File: posts/2014-11-14-ecosystem-vulnerability-tools.md (hunk 10 above)
Comment (Jin-Sun-tts) [reply posts/2014-11-14-ecosystem-vulnerability-tools.md]: fixed

File: posts/2014-11-14-ecosystem-vulnerability-tools.md (hunk 11 above)
Comment (Jin-Sun-tts) [reply posts/2014-11-14-ecosystem-vulnerability-tools.md]: fixed

File: posts/2014-11-14-ecosystem-vulnerability-tools.md (hunk 12 above)
Comment (Jin-Sun-tts) [reply posts/2014-11-14-ecosystem-vulnerability-tools.md]: fixed

File: posts/2014-11-14-ecosystem-vulnerability-tools.md (hunk 13 above)
Comment (Jin-Sun-tts) [reply posts/2014-11-14-ecosystem-vulnerability-tools.md]: fixed

**Comment by nickumia-reisys** [comment]
@btylerburton @Jin-Sun-tts Please take an extra close look at this.  I did not optimize code at all.  Kinda looking for help in including bootstrap more efficiently 🧊

File: _includes/layouts/tools.html [hunk 18]
```diff
@@ -0,0 +1,97 @@
...
+
+{% comment %}
+  This layout is used for individual posts / updates.
+
+  Default front matter is set in posts/posts.json. Individual
+  front matter and overrides of default values are available
+  in the front matter of specific posts in the posts directory.
+{% endcomment %}
+<!-- FOR MODAL FUNCTIONALITY - JavaScript Bundle with Popper -->
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.2/dist/css/bootstrap.min.css" integrity="sha384-xOolHFLEh07PJGoPkLv1IbcEPTNtaed2xpHsD9ESMhqIYd0nLMwNLD69Npy4HI+N" crossorigin="anonymous">
```
Comment (btylerburton) [inline _includes/layouts/tools.html:13]: Using a CDN for this should be fine.

File: _includes/layouts/tools.html [hunk 19]
```diff
@@ -0,0 +1,97 @@
...
+<script src="https://cdn.jsdelivr.net/npm/jquery@3.5.1/dist/jquery.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js" integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN" crossorigin="anonymous"></script>
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.2/dist/js/bootstrap.min.js" integrity="sha384-+sLIOodYLS7CIrQpBjl+C7nPvqq+FbNUBDunl/OZv93DB7Ln/533i8e/mZXLi/P+" crossorigin="anonymous"></script>
+<!-- FOR MODAL FUNCTIONALITY - JavaScript Bundle with Popper -->
+
+<!-- For Accessibility Checks -->
+<style>
+html > body > div > a, #menu-navigation > div > ul > li > a > span { color: #0072ed }
```
Comment (btylerburton) [inline _includes/layouts/tools.html:21]: are you adding these inline here because importing new stylesheets isn't working? if so, i'm working on that... 😃

**Review by Jin-Sun-tts — APPROVED** [review]

**Review by robert-bryson — APPROVED** [review]
Looks good. Great work!

**Pull request merged**